components/button/button.js

```diff
@@ -83,17 +90,19 @@ class Button extends ButtonMixin(LitElement) {
 				aria-label="${ifDefined(this.description || this.ariaLabel)}"
 				?autofocus="${this.autofocus}"
 				class="d2l-label-text"
-				?disabled="${this.disabled}"
+				?disabled="${this.disabled && !this.disabledTooltip}"
```

Comment:
Should we be wrapping the `click` event here and preventing bubble / preventing default if we're disabled?

Reply:
The mixin already stops the propagation of the click if `disabled`, so I _think_ that's handled for us.  I will add a unit test for this specific case.

Reply:
Ah ok, nice! I'm a little surprised TBH because the native `<button>` would have been doing that already when it's disabled. Hmm.